### PR TITLE
Update serverproperty-transact-sql.md

### PR DESCRIPTION
**Pull request by pwehland**
Synapse on-demand pool returns 11, not 6 like Synapse provisioned pool returns.

### CODE REVIEW

**Comment by PRMerger14** [comment]
@pwehland : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by VanMSFT** [comment]
Thanks @pwehland!

@julieMSFT - Hey Julie, can you please take a look? Thanks!

File: docs/t-sql/functions/serverproperty-transact-sql.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ Is an expression that contains the property information to be returned for the s
...
+|EngineEdition|[!INCLUDE[ssDE](../../includes/ssde-md.md)] edition of the instance of [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] installed on the server.<br /><br /> 1 = Personal or Desktop Engine (Not available in [!INCLUDE[ssVersion2005](../../includes/ssversion2005-md.md)] and later versions.)<br /><br /> 2 = Standard (This is returned for Standard, Web, and Business Intelligence.)<br /><br /> 3 = Enterprise (This is returned for Evaluation, Developer, and Enterprise editions.)<br /><br /> 4 = Express (This is returned for Express, Express with Tools, and Express with Advanced Services)<br /><br /> 5 = [!INCLUDE[ssSDS](../../includes/sssds-md.md)]<br /><br /> 6 = [!INCLUDE[ssDW](../../includes/ssdw-md.md)]<br /><br /> 8 = [!INCLUDE[ssSDSMIfull](../../includes/sssdsmifull-md.md)]<br /><br /> 9 = Azure SQL Edge (This is returned for all editions of Azure SQL Edge)<br /><br /> 11 = Azure Synapse Serverless (on-demand)<br /><br /> Base data type: **int**|
```
Comment (julieMSFT) [inline docs/t-sql/functions/serverproperty-transact-sql.md:62]: returned for all editions of Azure SQL Edge)<br /><br /> 11 = Azure Synapse serverless SQL pool<br /><br /> Base data type: **int**|  
```

**Review by julieMSFT — CHANGES_REQUESTED** [review]

**Review by julieMSFT — APPROVED** [review]

**Comment by julieMSFT** [comment]
#sign-off

**Comment by PRMerger20** [comment]
@julieMSFT: I'm sorry - only the author of this article, @VanMSFT, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by VanMSFT** [comment]
#sign-off

**Pull request merged**